src/pages/_app.tsx

```diff
@@ -9,6 +9,7 @@ import mdxComponents from 'components/mdx'
 import defaultSeoConfig from 'next-seo.json'
 import {useURL} from 'hooks/useUrl'
 import '../styles/index.css'
+import '@reach/listbox/styles.css'
```

Comment:
If I'm not mistaken NextJS should allow import css from node_modules on the component level, but I got this error:
![Image 2020-09-17 at 3 15 56 PM](https://user-images.githubusercontent.com/1519448/93469129-b88c4600-f8f8-11ea-93cf-2b50c264dede.png)
So I added css import to the `_app.tsx` for now

Reply:
Ya it doesn’t allow global style import at the component t level.